File: miranum-stack-showcase/order-example/order-example-core/src/main/java/io/miragon/orderExample/domain/Item.java
package io.miragon.orderExample.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Item
{
    private String name;
    private Integer quantity;
    private Double price;
}

File: taskmanagement/processes/order/src/main/kotlin/io/miragon/zeebe/tm/order/domain/Item.kt
package io.miragon.zeebe.tm.order.domain

import java.math.BigDecimal

data class Item(
    val id: String,
    val name: String? = null,
    val price: BigDecimal? = null,
    val image: String? = null,
    val quantity: Int? = null,
    val ready: Boolean? = null,
)